plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetConnectorTest.java

```diff
@@ -65,6 +65,36 @@ protected boolean supportsRowGroupStatistics(String typeName)
                 typeName.equalsIgnoreCase("timestamp(6) with time zone"));
     }
 
+    @Test
+    public void testCorrectnessForDelete()
```

Comment:
Do you have idea about the test name or cases @ebyhr @raunaqmorarka

Reply:
The right way to test this is through a unit test in `io.trino.parquet.reader.TestParquetReader`